app/assets/javascripts/discourse/tests/acceptance/user-menu-test.js

```diff
@@ -178,6 +186,190 @@ acceptance("User menu", function (needs) {
       "the tab's content is now displayed in the panel"
     );
   });
+
+  test("the profile tab", async function (assert) {
+    updateCurrentUser({ draft_count: 13 });
+    await visit("/");
+    await click(".d-header-icons .current-user");
+    await click("#user-menu-button-profile");
+
+    const summaryLink = query("#quick-access-profile ul li.summary a");
+    assert.ok(
+      summaryLink.href.endsWith("/u/eviltrout/summary"),
+      "has a link to the summary page of the user"
+    );
+    assert.strictEqual(
+      summaryLink.textContent.trim(),
+      I18n.t("user.summary.title"),
+      "summary link has the right label"
+    );
+    assert.ok(
+      summaryLink.querySelector(".d-icon-user"),
+      "summary link has the right icon"
+    );
+
+    const activityLink = query("#quick-access-profile ul li.activity a");
+    assert.ok(
+      activityLink.href.endsWith("/u/eviltrout/activity"),
+      "has a link to the activity page of the user"
+    );
+    assert.strictEqual(
+      activityLink.textContent.trim(),
+      I18n.t("user.activity_stream"),
+      "activity link has the right label"
+    );
+    assert.ok(
+      activityLink.querySelector(".d-icon-stream"),
+      "activity link has the right icon"
+    );
+
+    const invitesLink = query("#quick-access-profile ul li.invites a");
+    assert.ok(
+      invitesLink.href.endsWith("/u/eviltrout/invited"),
+      "has a link to the invites page of the user"
+    );
+    assert.strictEqual(
+      invitesLink.textContent.trim(),
+      I18n.t("user.invited.title"),
+      "invites link has the right label"
+    );
+    assert.ok(
+      invitesLink.querySelector(".d-icon-user-plus"),
+      "invites link has the right icon"
+    );
+
+    const dratsLink = query("#quick-access-profile ul li.drafts a");
+    assert.ok(
+      dratsLink.href.endsWith("/u/eviltrout/activity/drafts"),
+      "has a link to the drafts page of the user"
+    );
+    assert.strictEqual(
+      dratsLink.textContent.trim(),
+      I18n.t("drafts.label_with_count", { count: 13 }),
+      "drafts link has the right label with count of the user's drafts"
+    );
+    assert.ok(
+      dratsLink.querySelector(".d-icon-pencil-alt"),
+      "drafts link has the right icon"
+    );
+
+    const preferencesLink = query("#quick-access-profile ul li.preferences a");
+    assert.ok(
+      preferencesLink.href.endsWith("/u/eviltrout/preferences"),
+      "has a link to the preferences page of the user"
+    );
+    assert.strictEqual(
+      preferencesLink.textContent.trim(),
+      I18n.t("user.preferences"),
+      "preferences link has the right label"
+    );
+    assert.ok(
+      preferencesLink.querySelector(".d-icon-cog"),
+      "preferences link has the right icon"
+    );
```

Comment:
I know we're just maintaining parity but we should bring this up on dev as it kinds of overlaps with Sidebar.

Reply:
yep, will do when this is merged.